spec/coffee/helpers/google-api-mock.coffee

```diff
@@ -1,3 +1,6 @@
+capitalize = (s) ->
```

Comment:
Why not put this inside the factory, but outside the class definition so we're not polluting the global space?